src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.scss

```diff
@@ -188,95 +196,96 @@ $sidebar-width: 200px;
-        }
-
-        > .badge {
-          margin-left: 5px;
-        }
-
-        i.fa.fa-circle {
-          animation: blink 2s ease-in infinite;
-          font-size: 0.875em;
-          margin-top: 4px;
-          position: absolute;
-          right: 35px;
-        }
-
-        @keyframes blink {
-          from,
-          to {
-            opacity: 1;
-          }
-
-          50% {
-            opacity: 0;
-          }
-        }
-      }
-
-      li.active > a,
-      li > a a[aria-expanded='true'] {
-        color: vv.$white;
-      }
-    }
+    // ul {
+    //   &.component {
+    //     margin: 0;
+    //     padding: 20px 0;
+    //   }
+
+    //   p {
+    //     color: vv.$white;
+    //     padding: 10px;
+    //   }
+
+    //   li a {
+    //     color: vv.$white;
+    //     display: block;
+    //     font-size: 1.3em;
+    //     padding: 10px 23px 10px 10px;
+    //     text-decoration: none;
+
+    //     &:hover {
+    //       background: vv.$primary;
+    //       color: vv.$white;
+    //     }
+
+    //     > .badge {
+    //       margin-left: 5px;
+    //     }
+
+    //     i.fa.fa-circle {
+    //       animation: blink 2s ease-in infinite;
+    //       font-size: 0.875em;
+    //       margin-top: 4px;
+    //       position: absolute;
+    //       right: 35px;
+    //     }
+
+    //     @keyframes blink {
+    //       from,
+    //       to {
+    //         opacity: 1;
+    //       }
+
+    //       50% {
+    //         opacity: 0;
+    //       }
+    //     }
+    //   }
+
+    //   li.active > a,
+    //   li > a a[aria-expanded='true'] {
+    //     color: vv.$white;
+    //   }
+    // }
   }
 
-  a.dropdown-toggle {
-    position: relative;
-
-    &::after {
-      border: 0;
-      content: '\f054';
-      font-family: 'ForkAwesome';
-      font-size: 1rem;
-      margin-top: 2px;
-      position: absolute;
-      right: 20px;
-      transition: transform 0.3s ease-in-out;
-    }
-
-    &[aria-expanded='true']::after {
-      transform: rotate(90deg);
-    }
-  }
-
-  ul ul a {
-    background: lighten(vv.$secondary, 10);
-    font-size: 1.1em !important;
-    padding-left: 35px !important;
-  }
-
-  .cd-navbar-primary a:focus {
-    outline: none;
-  }
-
-  ngx-simplebar {
-    height: 100%;
-  }
+  // a.dropdown-toggle {
+  //   position: relative;
+
+  //   &::after {
+  //     border: 0;
+  //     content: '\f054';
+  //     font-family: 'ForkAwesome';
+  //     font-size: 1rem;
+  //     margin-top: 2px;
+  //     position: absolute;
+  //     right: 20px;
+  //     transition: transform 0.3s ease-in-out;
+  //   }
+
+  //   &[aria-expanded='true']::after {
+  //     transform: rotate(90deg);
+  //   }
+  // }
+
+  // ul ul a {
+  //   background: lighten(vv.$secondary, 10);
+  //   font-size: 1.1em !important;
+  //   padding-left: 35px !important;
+  // }
+
+  // .cd-navbar-primary a:focus {
+  //   outline: none;
+  // }
+
+  // ngx-simplebar {
+  //   height: 100%;
+  // }
```

Comment:
I am guessing these codes will be removed by the end of this PR? or are you planning to do a later PR for doing these cleanups?